lib/spoom/cli/commands/bump.rb

```diff
@@ -0,0 +1,85 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'find'
+require 'open3'
+
+require_relative 'base'
+
+module Spoom
+  module Cli
+    module Commands
+      class Bump < Base
+        extend T::Sig
+
+        default_task :bump
+
+        desc "bump", "bump Sorbet sigils from `false` to `true` when no errors"
+        sig { params(directory: String, extension: String).returns(T::Array[String]) }
+        def bump(directory = ".", extension = "rb")
+          files_to_bump = Bump.files_with_sigil_strictness(directory, "false", extension)
+
+          Bump.change_sigil_in_files(files_to_bump, "true")
+
+          output, no_errors = Spoom::Sorbet.srb_tc(File.expand_path(directory), capture_err: true)
+
+          return [] if no_errors
+
+          errors = Spoom::Sorbet::Errors::Parser.parse_string(output)
+
+          files_with_errors = Bump.file_names_from_error(errors)
```

Comment:
```suggestion
          files_with_errors = errors.map(&:file)
```

No need for a method